packages/constants/src/index.ts

```diff
@@ -1,3 +1,50 @@
-export const ZeroBytes32 = '0x0000000000000000000000000000000000000000000000000000000000000000';
-export const NativeAssetId = ZeroBytes32;
-export const EmptyRoot = '0xe3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855';
+import { bn } from '@fuel-ts/math';
+
+export const getEnv = () => ({
```

Comment:
what you think on changing this name to `getConstants`?

`getEnv` reminds me of env variables, which is not the case here

Reply:
`getConstants()` sounds good to me